### PR TITLE
DEP Use PHPUnit 11

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/305

### CODE REVIEW

File: tests/php/ContentReviewBaseTest.php [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-/**
- * Extend this class when writing unit tests which are compatible with other modules.
- * All compatibility code goes here.
- */
-abstract class ContentReviewBaseTest extends FunctionalTest
-{
-    /**
-     * @var bool
-     */
-    protected $translatableEnabledBefore;
```
Comment (emteknetnz) [inline tests/php/ContentReviewBaseTest.php:17]: This property is unused, this class did absolutely nothing

File: tests/php/SiteTreeContentReviewTest.php [hunk 2]
```diff
@@ -372,14 +373,30 @@ public function testPermissionCheckByOnDataObject()
...
-        // Mock Page class with canReviewContent method to return true on first call and false on second call
-        $mock = $this->getMockBuilder(Page::class)
-            ->setMethods(['canReviewContent', 'NextReviewDate', 'OwnerUsers'])
-            ->getMock();
-        $mock->expects($this->exactly(2))->method('canReviewContent')->willReturnOnConsecutiveCalls(false, true);
-        $mock->method('NextReviewDate')->willReturn('2020-02-20 12:00:00');
-        $mock->method('OwnerUsers')->willReturn(ArrayList::create([$reviewer]));
-        $mock->ContentReviewType = 'Custom';
+        // Mock Page class with canReviewContent method to return false on first call and true on second call
+        $mock = new class extends Page {
```
Comment (ChloeSartorelli) [inline tests/php/SiteTreeContentReviewTest.php]: Move this class into its own file, it's bigger than a couple of lines

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: tests/php/SiteTreeContentReviewTest.php (hunk 2 above)
Comment (emteknetnz) [reply tests/php/SiteTreeContentReviewTest.php]: Done

**Review by GuySartorelli — APPROVED** [review]
LGTM will merge when all other PRs are approved

**Pull request merged**